test/executor.test.js

```diff
@@ -52,6 +52,39 @@ describe('executor', function() {
     }
   });
 
+  describe('when booting', function() {
+    it('should set the booting status', function(done) {
+      expect(app.booting).to.be.undefined;
+      boot.execute(app, dummyInstructions, function(err) {
+        expect(err).to.be.undefined;
+        expect(app.booting).to.be.false;
+        done();
+      });
+    });
+
+    it('should emit the `booted` event', function(done) {
+      app.on('booted', function() {
+        expect(true).to.be.ok;
+        done();
+      });
+      boot.execute(app, dummyInstructions, function(err) {
+        expect(err).to.be.undefined;
+      });
+    });
+
+    it('should work when called asynchronously', function(done) {
+      boot.execute(app, dummyInstructions, function(err) {
+        expect(err).to.be.undefined;
+        done();
+      });
+    });
```

Comment:
What is the purpose of this test? Isn't it just duplicating the existing test "executor with boot with callback should run `boot/*` files asynchronously"?